presto-tests/src/test/java/com/facebook/presto/tests/TestQueryPlansDeterministic.java

```diff
@@ -82,4 +82,15 @@ public void testTpchQ9deterministic()
                 "  nation,\n" +
                 "  o_year DESC\n");
     }
+
+    @Test
+    public void testJoinsDeterministic()
+            throws Exception
+    {
+        determinismChecker.checkPlanIsDeterministic("" +
```

Comment:
removed the test as after rebasing on master it didn't catch any errors any more